### ISSUE TITLE
0.0.0.0 Day

### ISSUE THREAD

**Issue by oneofthedamons** [ACK-J/Port_Authority#38]
Does this add-on mitigate the [0.0.0.0 Day](https://www.oligo.security/blog/0-0-0-0-day-exploiting-localhost-apis-from-the-browser) issue?

**Comment by saibaaimo** [comment]
I don't think it does, see author's regex test case at https://regex101.com/r/DOPCdB/17 and code at https://github.com/ACK-J/Port_Authority/blob/main/background.js#L83
Shouldn't be difficult to patch it though.

**Comment by hackademix** [comment]
> Does this add-on mitigate the [0.0.0.0 Day](https://www.oligo.security/blog/0-0-0-0-day-exploiting-localhost-apis-from-the-browser) issue?

FWIW, [this](https://noscript) one (NoScript) [does](https://noscript.net/usage/#lan-protection).

**Comment by saibaaimo** [comment]
uBlock Origin easylist  was updated (https://github.com/easylist/easylist/commit/3b9d57fe) to guard against it https://github.com/uBlockOrigin/uAssets/issues/24867

**Comment by ACK-J** [comment]
Updated regex in v2.0.0 now accounts for 0.0.0.0
https://regex101.com/r/LSL180/1
